modules/options-for-notebook-server-environments.adoc

```diff
@@ -328,7 +328,7 @@ a| * Python 3.9
 [IMPORTANT] 
 ====
 *Disclaimer:* +
-{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench.
+{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms. You should review their licensing terms before you use this sample workbench.
```

Comment:
Not top priority right now, but the only thing I would suggest is removing the word "should".